inventory/s1-ha.casl.example.com.d/inventory/hosts

```diff
@@ -4,34 +4,45 @@
 # Create an OSEv3 group that contains the master, nodes, etcd, and lb groups.
 # The lb group lets Ansible configure HAProxy as the load balancing solution.
 # Comment lb out if your load balancer is pre-configured.
+[cluster_hosts:children]
+OSEv3
+dns
+
 [OSEv3:children]
 masters
 nodes
 etcd
+lb
 
 # Set variables common for all OSEv3 hosts
 #[OSEv3:vars]
 
 # For OSEv3 normal group vars, see ./group_vars/OSEv3.yml
 
 # Host Groups
+[lb:children]
+lb.s1-ha.casl.example.com
 
-[masters]
+[masters:children]
+masters.s1-ha.casl.example.com
 
-[etcd]
+[etcd:children]
+masters.s1-ha.casl.example.com
 
 [nodes:children]
 masters
-infra_hosts
-node_hosts
-
-[node_hosts]
-[node_hosts:vars]
-openshift_node_labels="{'region': 'primary', 'zone': 'default'}"
```

Comment:
@tomassedovic no, this change won't remove the node labels. it actually is just  removing this host group name, as it's no longer used. the node labels get set based on the instance metadata pulled from openstack.